### ISSUE TITLE
Add feature normalization to gradient descent + optimize learning rate & max iterations

### ISSUE THREAD

**Issue by wallacelim97** [AY1920S1-CS2103T-W12-3/main#143]

**Comment by wallacelim97** [comment]
Complete
